### DISCUSSION TITLE
Quickstart fails

### DISCUSSION THREAD

**Discussion by char-cole** [wpengine/faustjs#1842] [General]
### Description

Following this guide: https://faustjs.org/tutorial/get-started-with-faust

I'm not able to set up an example Faust project.

Using the given terminal command, I get the error:
`Could not locate the repository for "https://github.com/wpengine/faustjs/tree/main". Please check that the repository exists and try again.`

I also tried these and got the same result:
` npx create-next-app test-faust-next --use-npm --example https://github.com/wpengine/faustjs/tree/main/ `
`npx create-next-app test-faust-next --use-npm --example https://github.com/wpengine/faustjs/tree/main/examples/next/faustwp-getting-started`

Then I tried cloning the faustjs repo locally and running this:
`npx create-next-app test-faust-next --example  faustjs/examples/next/faustwp-getting-started/`
and got the following error:
`Could not locate an example named "faustjs/examples/next/faustwp-getting-started/"`

I'm using Node version 21.7.1 on macOS Sonoma 14.1.2. Was initially behind a company VPN but disabling that had no effect.

### Steps to reproduce

run 
```
npx create-next-app \
    -e https://github.com/wpengine/faustjs/tree/main \
    --example-path examples/next/faustwp-getting-started \
    --use-npm
```

### @faustwp/core Version

?

### @faustwp/cli Version

?

### FaustWP Plugin Version

?

### WordPress Version

?

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

**Comment by mindctrl** [comment]
> Could not locate the repository for "https://github.com/wpengine/faustjs/tree/main". Please check that the repository exists and try again.

@char-cole this sounds like a network issue. I just tried it and it's working as expected:

```
npx create-next-app \ \
    -e https://github.com/wpengine/faustjs/tree/main \
    --example-path examples/next/faustwp-getting-started \
    --use-npm
✔ What is your project named? … my-app
Creating a new Next.js app in /code/temp/test/my-app.

Downloading files from repo https://github.com/wpengine/faustjs/tree/main. This might take a moment.

Installing packages. This might take a couple of minutes.

....

Initialized a git repository.

Success! Created my-app at /code/temp/test/my-app
Inside that directory, you can run several commands:

  npm run dev
    Starts the development server.

  npm run build
    Builds the app for production.

  npm start
    Runs the built app in production mode.

We suggest that you begin by typing:

  cd my-app
  npm run dev

```

Can you try again now?

**Comment by char-cole** [comment]
thanks @mindctrl . I'm on a different network now and it still fails for me. I guess it's possible there's a security app on my computer blocking this operation.

How about using a locally cloned repo, is there a way to do that? I don't have experience with Nextjs either yet so I can't tell what it's looking for in an example

**Comment by mindctrl** [comment]
Hey @char-cole, interesting. Yes, you can clone the whole repo and then pull the Getting Started example from the `examples/next/faustwp-getting-started` directory.

**Comment by Vane-Toro** [comment]
I was having the same issue I updated the code to `--example https://github.com/wpengine/faustjs/tree/main \` and it worked.
